packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/types/model/FlutterModelFieldType.kt

```diff
@@ -29,7 +29,8 @@ data class FlutterModelFieldType(val map: Map<String, Any>) {
     private val ofModelName: String? = map["ofModelName"] as String?
 
     fun isEnum(): Boolean {
-        return fieldType == "enumeration"
+        return false;
+        //return fieldType == "enumeration"
```

Comment:
probably need to revert this change?

Reply:
spoke with Jameson, isEnum should always be false.